velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -127,6 +156,28 @@ class SubstraitVeloxPlanConverter {
       const ::substrait::Expression& substraitFilter,
       std::vector<::substrait::Expression_ScalarFunction>& scalarFunctions);
 
+  /// Helper function to extract the attributes required to create a ProjectNode
+  /// used for interpretting Substrait Emit.
+  EmitInfo GetEmitInfo(
+      const ::substrait::RelCommon& relCommon,
+      const core::PlanNodePtr& node) {
+    const auto& emit = relCommon.emit();
+    int emitSize = emit.output_mapping_size();
+    std::vector<core::TypedExprPtr> emitExpressions(emitSize);
```

Comment:
nit: Any particular reason not use use EmitInfo struct for these?

Reply:
I didn't catch it. We create the `EmitInfo` object from these values.

Reply:
I suggest to replace emitXxx variables with a single variable of type EmitInfo:

```
EmitInfo emitInfo;

for ...
      emitInfo.projectNames.push_back(outputType->nameOf(mapId));
      emitInfo.expressions[i].push_back(std::make_shared<core::FieldAccessTypedExpr>(
          outputType->childAt(mapId), outputType->nameOf(mapId)));

return emitInfo;
```

Reply:
Got it.

Reply:
I updated the PR.